### PR TITLE
Add SWAD dataset (updated)

### PR DESCRIPTION
**Pull request by yilingchung**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,6 +180,19 @@ Please send contributions via github pull request. You can do this by visiting t
 
 <a id="English-header"></a>
 ### English
+#### Do You Really Want to Hurt Me? Predicting Abusive Swearing in Social Media
+* Link to publication: [https://www.aclweb.org/anthology/2020.lrec-1.765.pdf](https://www.aclweb.org/anthology/2020.lrec-1.765.pdf)
+* Link to data: [https://github.com/dadangewp/SWAD-Repository](https://github.com/dadangewp/SWAD-Repository)
+* Task description: Binary (abusive swear word, non-abusive swear word)
+* Details of task: Abusive swearing
+* Size of dataset: 1,511
+* Percentage abusive: 0.41
```
Comment (leondz) [inline README.md:189]: Is this dataset 0.41% abusive, i.e. about four per thousand instances?

Comment (yilingchung) [reply README.md:189]: This annotation for this dataset is done on word level (swear word): 620 swear words were annotated as abusive out of 1511 swear words, so 0.41 % abusive. In their paper, they report it this way.
But if checking the dataset on the post level (one tweet per instance), there are 856 abusive tweet instances out of 1675 tweet instances (which is different from what is reported in the paper - 1320 ), so 0.51 % abusive. 

We can report both and state it clearly.

Comment (leondz) [reply README.md:189]: oh, that's a great catch! yes, good idea.

**Comment by yilingchung** [comment]
I updated the size of dataset and percentage abusive

**Review by leondz — APPROVED** [review]
lgtm

**Pull request merged**